go/vt/vtctl/workflow/traffic_switcher_test.go

```diff
@@ -91,12 +97,25 @@ func TestGetTargetSequenceMetadata(t *testing.T) {
 	env := newTestEnv(t, ctx, cell, sourceKeyspace, targetKeyspace)
 	defer env.close()
 
+	env.tmc.schema = map[string]*tabletmanagerdatapb.SchemaDefinition{
+		unescapedTable: {
+			TableDefinitions: []*tabletmanagerdatapb.TableDefinition{
+				{
+					Name:   unescapedTable,
+					Schema: tableDDL,
+				},
+			},
+		},
+	}
+
 	type testCase struct {
-		name          string
-		sourceVSchema *vschema.Keyspace
-		targetVSchema *vschema.Keyspace
-		want          map[string]*sequenceMetadata
-		err           string
+		name                           string
+		sourceVSchema                  *vschema.Keyspace
+		targetVSchema                  *vschema.Keyspace
+		options                        *vtctldatapb.WorkflowOptions
+		want                           map[string]*sequenceMetadata
+		expectSourceApplySchemaRequest *applySchemaRequestResponse
```

Comment:
there seems to be a mismatch between the field name and type. Should the field name also be `...Response`?

Reply:
It gets a big long, and this is test-only code where you can clearly see the type, but I can change it.

Reply:
oh you mean if you call it `...RequestResponse`? I think `expectApplySchemaResponse` is adequate.

Reply:
I changed it here: https://github.com/vitessio/vitess/pull/16860/commits/733198b0c7bbc7368a5ce007b28a87890fbb0f2e